understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                      </p>
                      
                      
-                     <p>It is possible to use visual patterns such as strips switching places to disguise
-                        a change of focus indicator. However, this is not considered a visible indicator.
+                     <p>短冊が入れ替わるような視覚的なパターンを使って、フォーカスインジケータの変更を偽装することは可能である。しかし、これは可視のインジケータとはみなされない。
                      </p>
                      
                   </section>
                   
                </section>
                <section id="relationship-with-non-text-contrast">
                   
-                  <h3>Relationship with Non-text Contrast</h3>
+                  <h3>非テキストのコントラストとの関係</h3>
                   
                   
-                  <p>Focus indicators are visual information required to identify a state of a user interface
-                     component. That means that they are subject to <a href="non-text-contrast" class="sc">1.4.11 Non-text Contrast</a>, in addition to 2.4.13 Focus Appearance.
+                  <p>フォーカスインジケータは、ユーザインタフェース コンポーネントの状態を特定するために必要な視覚情報である。つまり、2.4.13 フォーカスの外観に加え、<a href="non-text-contrast" class="sc">1.4.11 非テキストのコントラスト</a>が適用される。
                   </p>
                   
                   
-                  <p>In combination with <a href="focus-visible">2.4.7 Focus Visible</a>, <a href="non-text-contrast" class="sc">1.4.11 Non-text Contrast</a> requires that the visual focus indicator for a component must have sufficient contrast
-                     against the adjacent colors when the component is focused, except where the appearance
-                     of the component is determined by the user agent and not modified by the author.
+                  <p><a href="focus-visible">2.4.7 フォーカスの可視化</a>との組み合わせで、<a href="non-text-contrast" class="sc">1.4.11 非テキストのコントラスト</a>は、コンポーネントの外観がユーザエージェントによって決定され、コンテンツ制作者によって変更されない場合を除き、コンポーネントにフォーカスが当たったときに、コンポーネントの視覚的なフォーカスインジケータは、隣接する色に対して十分なコントラストを持たなければならないことを要求する。
                   </p>
                   
                   
-                  <p>The difference between the contrast requirements in Focus Appearance and Non-text
-                     Contrast is:
+                  <p>フォーカスの外観と非テキストのコントラスト要件の違いは、次のとおり:
                   </p>
                   
                   
                   <ul>
                      
-                     <li>Focus Appearance requires that focus indicators have a <em>change of contrast</em> between focused and non-focused states.
+                     <li>フォーカスの外観は、フォーカスインジケータが、フォーカスされた状態とフォーカスされていない状態の間で<em>コントラストが変化する</em>ことを要求する。
                      </li>
                      
-                     <li>Non-text Contrast requires that focus indicators have <em>adjacent contrast</em> between the indicator (in the focused state) and adjacent non-indicator colors.
+                     <li>非テキストコントラストでは、フォーカスインジケータは、インジケータ (フォーカス状態) と隣り合う非インジケータ色との間に<em>隣接コントラスト</em>を持つことが必要である。
```

Comment:
「非テキストコントラスト」は、達成基準名に合わせて「非テキストのコントラスト」としました。

```suggestion
                     <li>非テキストのコントラストでは、フォーカスインジケータは、インジケータ (フォーカス状態) と隣り合う非インジケータ色との間に<em>隣接コントラスト</em>を持つことが必要である。
```